src/XIVLauncher.Core/GameFixes/MacVideoFix.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using XIVLauncher.Common.Util;
+using XIVLauncher.Common.Unix.Compatibility.GameFixes;
+
+namespace XIVLauncher.Core.GameFixes;
+
+public class MacVideoFix : GameFix
+{
+    private const string MAC_ZIP_URL = "https://mac-dl.ffxiv.com/cw/finalfantasyxiv-1.1.2.zip";
```

Comment:
Should we have this point to somewhere in our repo (such as `assets/mac_fix_url`) so that we can update it without pushing an entire launcher update. It would make it easier for us and 3rd party packagers as if this breaks often we can fix it faster.